### ISSUE TITLE
Running tests from SQL's "security" project doesn't work anymore

### ISSUE THREAD

**Issue by astefan** [elastic/elasticsearch#41300]
This issue seems to be similar with https://github.com/elastic/elasticsearch/issues/41264. The PR mentioned there - https://github.com/elastic/elasticsearch/pull/40877 - seems to have changed the way tests are executed in this project.

At the moment, running `gradlew :x-pack:plugin:sql:qa:security:without-ssl:integTestRunner --tests "org.elasticsearch.xpack.sql.qa.security.JdbcSecurityIT.test*" -Dtests.seed=817215EC9743C1CF -Dtests.security.manager=true -Dtests.locale=bn-BD -Dtests.timezone=America/Denver -Dcompiler.java=12 -Druntime.java=11` there is no error but it **will not run any tests** from that integration test class.

Reverting back to the commit before the one mentioned above, and using `gradlew :x-pack:plugin:sql:qa:security:with-ssl:integTestRunner -Dtests.seed=CDAFA8CD10B79641 -Dtests.class=org.elasticsearch.xpack.sql.qa.security.JdbcSecurityIT -Dtests.method="test*" -Dtests.security.manager=true` everything seems ok.

Testing the same commands as above on the PR https://github.com/elastic/elasticsearch/pull/40877 will output an error message:

```
[2019-04-17T14:36:04,601][INFO ][o.e.n.Node               ] [node-0] starting ...                                                                                         
[2019-04-17T14:36:05,402][INFO ][o.e.t.TransportService   ] [node-0] publish_address {127.0.0.1:58381}, bound_addresses {127.0.0.1:58381}, {[::1]:58382}                  
[2019-04-17T14:36:05,413][WARN ][o.e.b.BootstrapChecks    ] [node-0] Java version [11-ea] is an early-access build, only use release builds                               
[2019-04-17T14:36:05,426][INFO ][o.e.c.c.Coordinator      ] [node-0] setting initial configuration to VotingConfiguration{jJn6jDtNTCunlgTvfWmEog}                         
[2019-04-17T14:36:05,425][WARN ][o.e.d.FileBasedSeedHostsProvider] [node-0] expected, but did not find, a dynamic hosts list at [W:\ECLIPS~3\ELASTI~1\x-pack\plugin\sql\qa
\security\WITHOU~1\build\cluster\INTEGT~1\ELASTI~1.0-S\config\unicast_hosts.txt]                                                                                          
[2019-04-17T14:36:05,566][INFO ][o.e.c.s.MasterService    ] [node-0] elected-as-master ([1] nodes joined)[{node-0}{jJn6jDtNTCunlgTvfWmEog}{G6JL9bfMRjS8z6Len4mucg}{127.0.0
.1}{127.0.0.1:58381}{xpack.installed=true, testattr=test} elect leader, _BECOME_MASTER_TASK_, _FINISH_ELECTION_], term: 1, version: 1, reason: master node changed {previo
us [], current [{node-0}{jJn6jDtNTCunlgTvfWmEog}{G6JL9bfMRjS8z6Len4mucg}{127.0.0.1}{127.0.0.1:58381}{xpack.installed=true, testattr=test}]}                               
[2019-04-17T14:36:05,647][INFO ][o.e.c.s.ClusterApplierService] [node-0] master node changed {previous [], current [{node-0}{jJn6jDtNTCunlgTvfWmEog}{G6JL9bfMRjS8z6Len4muc
g}{127.0.0.1}{127.0.0.1:58381}{xpack.installed=true, testattr=test}]}, term: 1, version: 1, reason: Publication{term=1, version=1}                                        
[2019-04-17T14:36:05,717][INFO ][o.e.g.GatewayService     ] [node-0] recovered [0] indices into cluster_state                                                             
=========================================                                                                                                                                 
                                                                                                                                                                          
FAILURE: Build failed with an exception.                                                                                                                                  
                                                                                                                                                                          
* What went wrong:                                                                                                                                                        
Execution failed for task ':x-pack:plugin:sql:qa:security:without-ssl:integTestRunner'.                                                                                   
> W:\Eclipse_workspace_master\elasticsearch\x-pack\plugin\sql\qa\security\without-ssl\build\classes\java\test does not exist.                                             
```

**Comment by astefan** [comment]
CC @atorok @mark-vieira

**Comment by mark-vieira** [comment]
This issues is resolved by #41321.
